statvar_imports/india_rbi/agriculture/test_data/sample_output/State_Wise_Area_Coarse_stat_vars.mcf

```diff
@@ -0,0 +1,10 @@
+# Auto generated using command: "stat_var_processor.py --input_data=/usr/local/google/home/chharish/IndiaRBI/data/statvar_imports/india_rbi/agriculture/test_data/sample_input/STATE-WISE AREA OF FOODGRAINS - COARSE CEREALS_data.xlsx --pv_map=/usr/local/google/home/chharish/IndiaRBI/data/statvar_imports/india_rbi/agriculture/Agriculture_pvmap.csv --config=/usr/local/google/home/chharish/IndiaRBI/data/statvar_imports/india_rbi/agriculture/Agriculture_config_metadata.csv --output_path=/usr/local/google/home/chharish/IndiaRBI/data/statvar_imports/india_rbi/agriculture/test_data/sample_output/State_Wise_Area_Coarse --places_resolved_csv=/usr/local/google/home/chharish/IndiaRBI/data/statvar_imports/india_rbi/agriculture/India_places_resolved.csv" on 2024-07-19 16:11:48.597109
```

Comment:
Please remove the commented line from everywhere?

Reply:
Removed those lines